shoplist/src/main/java/com/netguru/codereview/network/ShopListApiMock.kt

```diff
@@ -31,7 +31,7 @@ class ShopListApiMock : ShopListApi {
             }
         }
 
-    override fun getUpdateEvents(): Flow<String> = flow {
+    override fun getUpdateEvents(): Flow<String?> = flow {
```

Comment:
add Optional to the response of APIs, despite this being mocked, but when scale this code, it will be used as not-null, this could lead to error, prone, do not trust backend response.